src/models/outlets/outlets-request-dynamic-context-response.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface OutletsRequestDynamicContextResponse<T> {
```

Comment:
<T> is deprecated and replaced with <Type>, also it's not needed in this case since all the interface's properties are typed, and none of them is generic. Same comment applies on all the other interfaces in this PR.